docs/content/contributing.md

```diff
@@ -8,160 +8,26 @@ Thanks for your interest in contributing to the Open Policy Agent project!
-include a prefix on the first line of the commit message with the following
-format:
-
-```
-<package or directory path>: <description>
-```
-
-For example, a change to the `ast` package:
-```
-ast: Fix X when Y happens
-
-<Details...>
-
-Fixes: #123
-Signed-off-by: Random J Developer <random@developer.example.org>
-```
-
-or a change in the OPA website content (found in `./docs/content`):
-```
-docs/website: Add X to homepage for Y
-
-<Details...>
-
-Fixes: #456
-Signed-off-by: Random J Developer <random@developer.example.org>
-```
-
-### Developer Certificate Of Origin
-
-The OPA project requires that contributors sign off on changes submitted to OPA
-repositories.
-The [Developer Certificate of Origin (DCO)](https://developercertificate.org/)
-is a simple way to certify that you wrote or have the right to submit the code
-you are contributing to the project.
-
-The DCO is a standard requirement for Linux Foundation and CNCF projects.
-
-You sign-off by adding the following to your commit messages:
-
-    This is my commit message
-
-    Signed-off-by: Random J Developer <random@developer.example.org>
-
-Git has a `-s` command line option to do this automatically.
-
-    git commit -s -m 'This is my commit message'
-
-You can find the full text of the DCO here: https://developercertificate.org/
-
-### Code Review
-
-Before a Pull Request is merged, it will undergo code review from other members
-of the OPA community. In order to streamline the code review process, when
-amending your Pull Request in response to a review, do not squash your changes
-into relevant commits until it has been approved for merge. This allows the
-reviewer to see what changes are new and removes the need to wade through code
-that has not been modified to search for a small change.
-
-When adding temporary patches in response to review comments, consider
-formatting the message subject like one of the following:
+If you want to contribute code and you are new to the Go programming language, check out
+the [development reference](../contrib-development/) for help getting started.
 
-- `Fixup into commit <commit ID> (squash before merge)`
-- `Fixed changes requested by @username (squash before merge)`
-- `Amended <description> (squash before merge)`
+## Other types of contributions
 
-The purpose of these formats is to provide some context into the reason the
-temporary commit exists, and to label it as needing squashed before a merge
-is performed.
+We welcome contributions of all kinds. For example:
 
-It is worth noting that not all changes need be squashed before a merge is
-performed. Some changes made as a result of review stand well on their own,
-independent of other commits in the series. Such changes should be made into
-their own commit and added to the PR.
+- [Documentation](../contrib-docs) including reference material and examples.
+- Help out other members in the OPA Slack and GitHub channels
```

Comment:
Maybe call out the discussions board specifically? Also Stackoverflow, it's a relevant venue still, I guess.